### PR TITLE
Sukihanabi1789

### PR DESCRIPTION
**Pull request by sukihanabi**
draft

### CODE REVIEW

File: Skype/SfbOnline/what-are-calling-plans-in-office-365/change-the-emergency-location-for-a-user.md [hunk 1]
```diff
@@ -67,5 +67,9 @@ To learn how to get Calling Plans in Office 365 and how much they cost, see [Sky
 
 [Skype for Business Online: Emergency Calling disclaimer label](https://github.com/MicrosoftDocs/OfficeDocs-SkypeForBusiness/blob/live/Teams/downloads/emergency-calling/emergency-calling-label-(en-us)-(v.1.0).zip?raw=true)
 
+## Got feedback? 
+Skype for Business UserVoice is your place to suggest the features you’d like to see us add to Skype for Business. While we can’t guarantee any specific features or timelines, we will respond to every suggestion that gets at least 500 votes.
+Go to the [Skype for Business UserVoice](https://www.skypefeedback.com/forums/299913-generally-available/suggestions/18440797-powershell-emergency-location-telephone-number-l) .
+
```
Comment (SerdarSoysal) [inline Skype/SfbOnline/what-are-calling-plans-in-office-365/change-the-emergency-location-for-a-user.md]: Not approving this because the product button feedback below already goes to user voice. Do not make this change.

File: Skype/SfbOnline/set-up-skype-for-business-online/allow-users-to-contact-external-skype-for-business-users.md [hunk 2]
```diff
@@ -46,33 +46,36 @@ Use the steps in this article when:
...
     
     For a list of all ports you need to open, see [Office 365 URLs and IP address ranges](https://docs.microsoft.com/microsoftteams/office-365-urls-ip-address-ranges).
 
-8. Make sure that the administrator in the organization has also followed these steps.
+9. Make sure that the administrator in the organization has also followed these steps.
     
-9. **WAIT UP TO 24 HOURS TO TEST**. Any time you change the external communications settings, it can take up to 24 hours for the changes to populate across all the data centers.
+10. **WAIT UP TO 24 HOURS TO TEST**. Any time you change the external communications settings, it can take up to 24 hours for the changes to populate across all the data centers.
     
 ![Skype](../images/58550720-2a68-42d1-a926-1884e6aeb55c.png) You can allow your users to search for and IM with everyone who uses Skype, the free consumer app! To learn more, see [Let Skype for Business users add Skype contacts](let-skype-for-business-users-add-skype-contacts.md).
```
Comment (SerdarSoysal) [inline Skype/SfbOnline/set-up-skype-for-business-online/allow-users-to-contact-external-skype-for-business-users.md]: Approving this change

File: Skype/SfbServer/plan-your-deployment/clients-and-devices/mobile-feature-comparison.md [hunk 3]
```diff
@@ -172,8 +172,10 @@ On iOS devices, Skype for Business signs out automatically after the mobile clie
  &#x2776;  For Office 365 users, this feature requires Enterprise Voice, which is part of the E5 license.
   
  &#x2777;  Requires a WiFi connection by default.
+ 
+ &#x2778; Viewing embedded video in PowerPoint presentations is not supported.
```
Comment (SerdarSoysal) [inline Skype/SfbServer/plan-your-deployment/clients-and-devices/mobile-feature-comparison.md]: I see that you're adding a third footnote, but why remove the second one? (x2777)?  Probably should keep that in too, no?

File: Skype/SfbServer/plan-your-deployment/clients-and-devices/mobile-feature-comparison.md [hunk 4]
```diff
@@ -172,8 +172,10 @@ On iOS devices, Skype for Business signs out automatically after the mobile clie
...
   
-## Telephony support
+## Telephony Support
```
Comment (SerdarSoysal) [inline Skype/SfbServer/plan-your-deployment/clients-and-devices/mobile-feature-comparison.md]: Rejecting this change because we use sentence casing in section headers. It has to remain "Telephony support"

File: Teams/Exchange-Teams-interact.md [hunk 5]
```diff
@@ -63,6 +63,8 @@ Additional information:
 
 -   For Exchange on-premises, you must configure the new Exchange OAuth authentication protocol as described in [Configure OAuth authentication between Exchange and Exchange Online organizations](https://technet.microsoft.com/en-us/library/dn594521(v=exchg.150).aspx). 
 
+- Control and protect the configuration of compliance policies in Teams and Exchange using Conditional Access. For more information see [How do Conditional Access policies work for Teams?](https://docs.microsoft.com/en-us/microsoftteams/security-compliance-overview#how-do-conditional-access-policies-work-for-teams) and [Set up app-based conditional access policies with Intune](https://docs.microsoft.com/en-us/intune/app-based-conditional-access-intune-create)
+
 > [!NOTE]
```
Comment (SerdarSoysal) [inline Teams/Exchange-Teams-interact.md]: First link is ok, but I would not add the link to Intune.

File: Teams/audio-conferencing-pay-per-minute.md [hunk 6]
```diff
@@ -51,6 +51,20 @@ Whereas the Audio Conferencing per-user license offer includes dial-in usage and
 Pay-per-minute requires your organization to have [Communications Credits](what-are-communications-credits.md) enabled with a license assigned to each user in order for Audio Conferencing to work. If you want more details, see [Set up Communications Credits for your organization](set-up-communications-credits-for-your-organization.md) and/or [Microsoft Teams add-on licensing](teams-add-on-licensing/microsoft-teams-add-on-licensing.md).
   
 To enable Audio Conferencing pay-per-minute for users in your organization, see [Try or purchase Audio Conferencing in Office 365](try-or-purchase-audio-conferencing-in-office-365-for-teams.md).
+
+## Why is it best for you?
+
+- Pay per-minute will only be charged on a per-minute basis for each inbound or outbound call placed by each attendee during a scheduled meeting (rates vary for toll or toll-free call and by destination) instead of using Skype for Business or Teams application in a mobile device or PC.
+
```
Comment (SerdarSoysal) [inline Teams/audio-conferencing-pay-per-minute.md]: This comment isn't clear.

File: Teams/audio-conferencing-pay-per-minute.md [hunk 7]
```diff
@@ -51,6 +51,20 @@ Whereas the Audio Conferencing per-user license offer includes dial-in usage and
 Pay-per-minute requires your organization to have [Communications Credits](what-are-communications-credits.md) enabled with a license assigned to each user in order for Audio Conferencing to work. If you want more details, see [Set up Communications Credits for your organization](set-up-communications-credits-for-your-organization.md) and/or [Microsoft Teams add-on licensing](teams-add-on-licensing/microsoft-teams-add-on-licensing.md).
```
Comment (SerdarSoysal) [inline Teams/audio-conferencing-pay-per-minute.md]: Need to understand better where this information is coming from. It is making a lot of assertions which will need to be vetted by the Product and marketing teams. Recommend submitting the changes in this file as a separate pull request.

File: Teams/create-an-org-wide-team.md [hunk 8]
```diff
@@ -23,7 +23,11 @@ Org-wide teams provide an automatic way for everyone in a small to medium-sized
  
 With org-wide teams, global administrators can easily create a public team that pulls in every user in the organization and keeps the membership up to date with Active Directory as users join and leave the organization. Only global admins can create org-wide teams and currently an org-wide team is limited to organizations with no more than 2,500 users. If these requirements are met, global admins will see **Org-wide** as an option under **Privacy** when creating a team. 
 
-![Screen shot of the Org-wide option to create an org-wide team](media/create-org-wide-team.png "Screen shot of the Org-wide option to create an org-wide team")
+###### Create a Team
```
Comment (SerdarSoysal) [inline Teams/create-an-org-wide-team.md]: Instead of using headers, suggest using a numbered list.

File: Teams/create-an-org-wide-team.md [hunk 9]
```diff
@@ -23,7 +23,11 @@ Org-wide teams provide an automatic way for everyone in a small to medium-sized
...
+![Screen shot of the create a team](../Teams/CreateATeamSite.png)
+
```
Comment (SerdarSoysal) [inline Teams/create-an-org-wide-team.md]: the images must be moved to the media folder and the names need to be changed to adhere to the naming standards.

File: Teams/create-an-org-wide-team.md [hunk 10]
```diff
@@ -23,7 +23,11 @@ Org-wide teams provide an automatic way for everyone in a small to medium-sized
...
+###### Choose Org-wide
+![Screen shot of the Org-wide option to create an org-wide team](../Teams/OrgwideTeams.png)
```
Comment (SerdarSoysal) [inline Teams/create-an-org-wide-team.md]: See same comment above about image locations

**Comment by SerdarSoysal** [comment]
- Can't see the images (MSTeamsAdmin.png and SFBlegacy-size65.png) you are adding. Won't approve this PR without seeing the images.
- CreateATeam.PNG and CreateATeamSite.png must go in the Teams\Media folder. Please move those images there if you want to add them. Also, follow the naming convention that is there (for example: Create-a-team.png)

File: Teams/create-an-org-wide-team.md [hunk 11]
```diff
@@ -32,6 +36,7 @@ When an org-wide team is created, all global admins are added as team owners and
 
 > [!NOTE]
 > Rooms that aren't a part of a room list, equipment, and resource accounts might be added or synced to the org-wide team. Team owners can easily remove these accounts from the team.
+> Only users that has teams license will be added automatically in Org-Web Team, after the org-web team has been created the New Users and newly enabled licensed users will have the option to join Org-web once they login to Teams.
```
Comment (SerdarSoysal) [inline Teams/create-an-org-wide-team.md]: Should say "Only users that have a Teams license will be..."
Also, it should be "org-wide" team and not "Org-Web" team.
"New Users" should not be capitalized.
Instead of saying "newly enabled licensed users" say "users who get assigned licenses after the team is created"

File: Teams/create-an-org-wide-team.md [hunk 12]
```diff
@@ -58,3 +63,9 @@ If this isn't an option, you can create a public team using PowerShell and add a
 
 Global admins can convert an existing team to an org-wide team editing it in Teams client.
 Go to the team name and click More options ...(ellipsis) > Edit team.
+
+### Can we exclude groups or users not to be added as a member of an org-wide team?
+Since Org-wide channel is an automatic way for everyone in an organization to be a part of a team. It is best practice to check group membership after the team is created . Add members that should be in the group then remove members that doesn’t belong to the group.
+
+## Got feedback?
```
Comment (SerdarSoysal) [inline Teams/create-an-org-wide-team.md]: Remove uservoice reference.

File: Teams/create-an-org-wide-team.md [hunk 13]
```diff
@@ -58,3 +63,9 @@ If this isn't an option, you can create a public team using PowerShell and add a
 
 Global admins can convert an existing team to an org-wide team editing it in Teams client.
 Go to the team name and click More options ...(ellipsis) > Edit team.
+
+### Can we exclude groups or users not to be added as a member of an org-wide team?
+Since Org-wide channel is an automatic way for everyone in an organization to be a part of a team. It is best practice to check group membership after the team is created . Add members that should be in the group then remove members that doesn’t belong to the group.
```
Comment (SerdarSoysal) [inline Teams/create-an-org-wide-team.md]: Suggest revising this substantially. I would say that if you need to exclude groups/users, then you should create a specific team and not rely on the org-wide setting.

File: Teams/direct-routing-border-controllers.md [hunk 14]
```diff
@@ -34,7 +34,7 @@ With each vendor, Microsoft:
 The table below lists devices certified for Direct Routing. 
 
 [Learn more about Direct Routing](https://aka.ms/dr). 
-If you have any questions about SBC certification program for Direct Routing please email to drsbccertification@microsoft.com
+**If you have any questions about SBC certification program for Direct Routing please email to drsbccertification@microsoft.com**
```
Comment (SerdarSoysal) [inline Teams/direct-routing-border-controllers.md]: Rejecting format change. No need to bold this.

File: Teams/direct-routing-border-controllers.md [hunk 15]
```diff
@@ -44,7 +44,7 @@ If you have any questions about SBC certification program for Direct Routing ple
 |                                                                                                                     |  Mediant 2600 SBC   |     &#10004;     |   &#10004;    |  7.20A.250.003   |
 |                                                                                                                     |  Mediant 4000 SBC   |     &#10004;     |   &#10004;     |  7.20A.250.003   |
 |                                                                                                                     | Mediant 1000B  SBC  |     &#10004;     |   &#10004;     |  7.20A.250.003   |
-|                                                                                                                     | Virtual Edition SBC |     &#10004;     |   &#10004;     |  7.20A.250.003  |
+|                                                                                                                     | Virtual Edition SBC |     &#10004;     |   &#10004;     |  7.20A.250.003  
 |  [Ribbon Communications](https://ribboncommunications.com/solutions/enterprise-solutions/microsoft-skype-business)  |      SBC 5110       |     &#10004;     |   &#10004;    |       V6.2       |
```
Comment (SerdarSoysal) [inline Teams/direct-routing-border-controllers.md]: Rejecting this change. No need to modify the table structure. Removing that "|" will break the table when the topic is rendered on docs.

File: Teams/set-up-communications-credits-for-your-organization.md [hunk 16]
```diff
@@ -70,15 +70,19 @@ For more information, see [Microsoft Teams add-on licensing](teams-add-on-licens
...
   
-When you are setting up Communications Credits, you will need to investigate call usage for your organization to determine the amounts that you will need to put in. You can get call usage information by reviewing the **PSTN usage details** report. This report lets you export the call data records to Excel if you want to export the data for storage or create custom reports. To see usage, see [Skype for Business PSTN usage report](/SkypeForBusiness/skype-for-business-online-reporting/pstn-usage-report)
+When you are setting up Communications Credits, you will need to investigate call usage for your organization to determine the amounts that you will need to put in. You can get call usage information by reviewing the **PSTN usage details** report. This report lets you export the call data records to Excel if you want to export the data for storage or create custom reports. To see usage, see [Skype for Business PSTN usage report](https://docs.microsoft.com/en-us/skypeforbusiness/skype-for-business-online-reporting/pstn-usage-report)
```
Comment (SerdarSoysal) [inline Teams/set-up-communications-credits-for-your-organization.md]: Rejecting this change because there is no reason to change the relative link to a hard-coded link with the locale in it. It breaks localization.

File: Teams/set-up-communications-credits-for-your-organization.md [hunk 17]
```diff
@@ -70,15 +70,19 @@ For more information, see [Microsoft Teams add-on licensing](teams-add-on-licens
    - **Trigger amount** Enter the amount in **When the balance falls below** box that will be used to ' *trigger*  ' the auto-recharge. Once your balance falls below this amount, the recharge amount will be added automatically to your account.
 
       > [!NOTE]
+    
      > Funds will be applied only to Communications Credits at Microsoft published rates when the services are used. Any funds not used within 12 months of the purchase date will expire and be forfeited. 
+     >
+     > Monthly billing for Communication Credits will only be applied if the alloted fund has been used, to check on your monthly usage we recommend to view [Skype for Business PSTN usage report](https://docs.microsoft.com/en-us/skypeforbusiness/skype-for-business-online-reporting/pstn-usage-report)
```
Comment (SerdarSoysal) [inline Teams/set-up-communications-credits-for-your-organization.md]: This change is approved.

File: Teams/teams-live-events/plan-for-teams-live-events.md [hunk 18]
```diff
@@ -23,24 +23,33 @@ When you are planning Teams live events to hold large meetings in your organizat
 ## Who can create and schedule live events?
```
Comment (SerdarSoysal) [inline Teams/teams-live-events/plan-for-teams-live-events.md]: All the changes to this file must be approved by marketing. Please submit this particular change as a separate PR.

**Comment by SerdarSoysal** [comment]
@sukihanabi: we cannot merge this PR as is. I have made several comments in the files, approved some of the changes, rejected others, and made suggestions. Please review and adjust accordingly. 

Also, going forward, please refrain from combining multiple changes into a single PR. For example, the image changes in the org-wide teams creation should be a PR on its own. Same thing for communication credits changes. That would make processing these a lot faster because different people need to review and sign-off on the changes as different areas are owned by different writers.

Please contact @kenwith if you have follow-up questions.

**Comment by sukihanabi** [comment]
closing this PR will create One Issue - One PR
